gantry/firmware/spi.c

```diff
@@ -120,4 +122,23 @@ void Reset_CS_Pin() { HAL_GPIO_WritePin(GPIOB, GPIO_PIN_12, GPIO_PIN_RESET); }
 void hal_transmit_receive(uint8_t* transmit, uint8_t* receive,
                           uint16_t buff_size, uint32_t timeout) {
     HAL_SPI_TransmitReceive(&handle, transmit, receive, buff_size, timeout);
+}
+
+void Gantry_Driver_CLK_init(uint8_t gantry_axis) {
+    switch (gantry_axis) {
+        case 0x30:
```

Comment:
Rather than doing this, and having the exact value of the node id be what matters, can we have a symbolic value that we check? like define `GANTRY_X` or `GANTRY_Y` or something and use that to define this and the node id at the same time? Because now, if we ever change the node id value, this will silently break at runtime.